### DISCUSSION TITLE
添加存储成功后，打开页面显示failed get objs: failed to list objs

### DISCUSSION THREAD

**Discussion by everiori** [AlistGo/alist#3335] [Q&A]
本地用docker安装alist，添加小雅啥的都没问题，添加一个Alist V3的存储 https://106.52.245.210/ （直接访问该link可以打开对应的alist页面），然后在主页打开对应的目录后，显示如下错误，简单查了一下，是证书问题，但是不知道如何解决，请大佬们帮忙看一下。

failed get objs: failed to list objs: Post "https://************10/api/fs/list": x509: cannot validate certificate for ************10 because it doesn't contain any IP SANs

**Comment by Windlien** [comment]
I have the same problem.

**Reply by 695975525** [reply]
看一下安装环境的时间是不是对的，然后检查一下etc/ssl/certs目录下是不是有ca-certificates.crt，有的话复制到tmp目录下，
然后环境下看一下是不是装了其他的，比如aliyunwebdav、caddy、chfs之类的管理插件

**Reply by everiori** [reply]
感谢。都检查了一遍，docker安装的alist，时区是UTC，时间换算之后能对上，对应目录也是自动生成的，没改过，也没装过其他程序

**Comment by flydo** [comment]
刚刚添加189网盘时，也出现这个问题了，经查，是它自带的接口 ssl 出错（浏览器打开时没报错，但用 curl 报错了）。
```bash
# curl -vvv  https://api.nn.ci/ocr/file/json
*   Trying 45.66.130.152:443...
* Connected to api.nn.ci (45.66.130.152) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (OUT), TLS alert, unknown CA (560):
* SSL certificate problem: unable to get local issuer certificate
* Closing connection 0
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

按官方的文档，搭建一个自己的 ocr 就解决了：https://alist.nn.ci/zh/config/global.html#ocr-api

可以将下面的内容添加到 alist 同个 docker-compose.yml 里
```yaml
  ocr:
    image: xhofe/ddddocr_server:main
    container_name: ocr
    restart: unless-stopped
```

然后配置下下：   
后台->设置->全局->Ocr 接口。设置为以下网址
```
http://ocr:9898/ocr/file/json
```

**Comment by AlvinCR** [comment]
我这边禁用存储后重新开启，成功解决问题。报的错是一样的

**Comment by jiakjiang** [comment]
楼主解决了吗？我现在也是这个问题

**Comment by wieniawski-lai** [comment]
移动云,有着一样的问题
failed get objs: failed to list objs: 用户不存在

**Comment by languang666** [comment]
移动云,有着一样的问题
failed get objs: failed to list objs: 用户不存在

**Comment by sorakiseki24** [comment]
同问，移动云出现同样的问题，怎么解决

**Comment by XYK-CHINA** [comment]
我用的是VIP优盘的webdav，挂载后存储显示work，当我打开挂载的文件夹时就会显示failed get objs: failed to list objs:
